src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/LoadBalancerTests.cs

```diff
@@ -37,6 +37,14 @@ public void TestLoadBalancerCRUDPublic()
             NetworkResourcesController.NewInstance.RunPsTest(_logger, "Test-LoadBalancerCRUD-Public");
         }
 
+        [Fact]
```

Comment:
Are there cases that the non-generated tests look at the the generated tests look at? Otherwise there is no reason to have double the number of tests (will just make CI run slower for no reason).

Reply:
@maddieclayton generated and non-generated tests do overlap to some degree, but I don't think they do that enough to be concerned. Non-generated tests, for example, use piping and AsJob while generated tests rely on simple assigments.